src/Automation/Automation/Properties/Resources.resx

```diff
@@ -490,4 +490,7 @@
     <value>Removing Azure Automation Software Update Configuration.</value>
     <comment>Automation</comment>
   </data>
+  <data name="SoftwareUpdateConfigurationInvalidTagFormat" xml:space="preserve">
+    <value>The Tag format is invalid. Valid Tag format is hashtable with string key and value of String or array of string.</value>
+  </data>
 </root>
```

Comment:
@jemex (I can't comment on the file itself) why was the below `PS-VSPrompt.lnk` file changed?

Reply:
@cormacpayne  I didn't make any intentional change to this file. I will revert the file